build.gradle

```diff
@@ -166,25 +192,38 @@ static void assertExecutableSpecRequirementsForTests(Project project, Test test)
     test.systemProperties['concordion.output.dir'] = project.reporting.baseDir.getAbsolutePath() + "/spec"
     test.systemProperties['concordion.run.threadCount'] = '2.5C' // 2.5 * number of cores
 
-    def propertiesToMap = ['CT_PROJECT_KEY',
-                           'CT_CLIENT_ID',
-                           'CT_CLIENT_SECRET',
-                           'CT_PAYONE_INTEGRATION_URL',
+    def propertiesToMap = ['TEST_DATA_CT_PROJECT_KEY',
+                           'TEST_DATA_CT_CLIENT_ID',
+                           'TEST_DATA_CT_CLIENT_SECRET',
+                           'TEST_DATA_CT_PAYONE_INTEGRATION_URL',
+
+                           'TEST_DATA_TENANT_NAME',
 
                            'TEST_DATA_VISA_CREDIT_CARD_NO_3DS',
                            'TEST_DATA_VISA_CREDIT_CARD_3DS',
                            'TEST_DATA_3_DS_PASSWORD',
                            'TEST_DATA_SW_BANK_TRANSFER_IBAN',
                            'TEST_DATA_SW_BANK_TRANSFER_BIC',
+                           'TEST_DATA_SW_BANK_TRANSFER_PIN',
+                           'TEST_DATA_SW_BANK_TRANSFER_TAN',
 
                            'TEST_DATA_PAYONE_MERCHANT_ID',
                            'TEST_DATA_PAYONE_SUBACC_ID',
                            'TEST_DATA_PAYONE_PORTAL_ID',
-                           'TEST_DATA_PAYONE_KEY'
+                           'TEST_DATA_PAYONE_KEY',
+
+                           'TEST_DATA_TENANT_NAME_2',
+                           'TEST_DATA_CT_PROJECT_KEY_2',
+                           'TEST_DATA_CT_CLIENT_ID_2',
+                           'TEST_DATA_CT_CLIENT_SECRET_2',
```

Comment:
[`BaseTenant2Fixture`](https://github.com/commercetools/commercetools-payone-integration/blob/954f1460d4bd2fa027f3e342913cd97eb081d452/functionaltests/src/test/java/specs/multitenancy/BaseTenant2Fixture.java#L23-L23) explains how they are used.

i put it as a comment 426ef3e